CHANGELOG.adoc

```diff
@@ -80,6 +80,7 @@ This reduces the time required to build SysON, especially when using Spring Tool
 - https://github.com/eclipse-syson/syson/issues/790[#790] [interconnection-view] Add `New Interface` tool on `PartUsage` in Interconnection View diagram.
 - https://github.com/eclipse-syson/syson/issues/799[#799] [export] Handle `UseCaseUsage` and `IncludeUseCaseUsage` elements.
 - https://github.com/eclipse-syson/syson/issues/803[#803] [libraries] All standard libraries have been updated to comply with the SysML Beta 2.2 specification.
+- https://github.com/eclipse-syson/syson/issues/805[#805] [import] Improve non regression tests. Added source sysml file and unaltered ast.json result.
```

Comment:
Could you please add a new line between the two sentences?
We let only one sentence per line.

Reply:
done.